src/keeper/contracts/conditions/NFTs/NFTUpgradable.ts

```diff
@@ -103,7 +103,7 @@ export class NFTUpgradeable extends ContractBase {
      * @returns
      */
     public async balance(address: string, did: string): Promise<BigNumber> {
-        return this.call('balanceOf', [zeroX(address), didZeroX(did)])
+        return BigNumber.from(await this.call('balanceOf', [zeroX(address), didZeroX(did)]) || 0)
```

Comment:
I would remove `|| 0`. the call needs to return a number

Reply:
yeah, I just added it because I forgot to add `await` first